shacl12-node-expr/index.html

```diff
@@ -2647,6 +2647,153 @@ <h3>sh:nodeByExpression</h3>
 			
 		</section>
 
+		<section id="dynamic-shacl">
+			<h2>Dynamic SHACL</h2>
+			<p>
+				This section defines <dfn>Dynamic SHACL</dfn> as a dialect of SHACL that some implementations MAY support.
+			</p>
+			<p>
+				In Dynamic SHACL any <a>parameter</a> of a <a>constraint</a> can be computed using a <a>node expression</a>,
+				excluding those that do <em>not</em> allow <a>blank nodes</a> (such as <code>sh:node</code>)
+				but including those that take <a>SHACL lists</a> as values (such as <code>sh:class</code>,
+				<code>sh:datatype</code>, and <code>sh:in</code>).
+				During <a>validation</a>, such <a>node expressions</a> are evaluated in the <a>data graph</a>,
+				using the current <a>focus node</a>.
+				The resulting <a>nodes</a> will be used as <a>parameters</a> for the <a>constraint</a>.
+			</p>
+			<section class="informative">
+				<h3>Example: Dynamic Minimum Age of Presidents</h3>
+				<p>
+					As a use case of <a>Dynamic SHACL</a>, assume we want to express that the legal minimum age
+					of a president is 18 unless the country is USA, where it is 35.
+				</p>
+				<aside class="example" title="Example of Dynamic SHACL using a node expression at sh:minInclusive">
+					<div class="shapes-graph">
+						<div class="turtle">
+ex:PresidentShape
+    a sh:NodeShape ;
+	sh:targetClass ex:President ;
+    sh:property ex:PresidentShape-age ;
+.
+ex:PresidentShape-age
+    a sh:PropertyShape ;
+    sh:path ex:age ;
+    <b>sh:minInclusive [
+        shnex:if [
+            sparql:eq (
+                [ shnex:path ex:country ]
+                ex:USA
+            )
+        ]
+        shnex:then 35 ;
+        shnex:else 18 ;
+    ]</b> .
+						</div>
+						<div class="jsonld">
+							<pre class="jsonld">
+								TODO
+							</pre>
+						</div>
+					</div>
+				</aside>
+			</section>
+			<section class="informative">
+				<h3>Example: Dynamic Enumerations</h3>
+				<p>
+					As a use case of <a>Dynamic SHACL</a>, assume the following <a>data graph</a>.
+				</p>
+				<aside class="example" title="Example data graph with addresses">
+					<div class="data-graph">
+						<div class="turtle">
+ex:ArizonaAddress1
+    a ex:Address ;
+    ex:street "123 John Muir Ave" ;
+    ex:country ex:USA ;
+    ex:state "AZ" ;
+.
+ex:QueenslandAddress1
+    a ex:Address ;
+    ex:street "123 Bob Katter Cl" ;
+    ex:country ex:Australia ;
+    ex:state "QLD" ;
+.
+						</div>
+						<div class="jsonld">
+							<pre class="jsonld">
+								TODO
+							</pre>
+						</div>
+					</div>
+				</aside>
+				<p>
+					We want to express that the valid values of <code>ex:state</code> depend on the value of <code>ex:country</code>
+					at the given focus node.
+					For example, the valid values for country <code>ex:USA</code> would be <code>( "AL" "AK" "AZ" ... )</code>
+					while values for country <code>ex:Australia</code> would be <code>( "ACT" "NSW" "NT" "QLD" "SA" "TAS" "VIC" "WA" )</code>.
```

Comment:
Keeping with the surrounding content, this change should have been as below:
```suggestion
					while valid values for country <code>ex:Australia</code> would be <code>( "ACT" "NSW" "NT" "QLD" "SA" "TAS" "VIC" "WA" )</code>.
```